### PR TITLE
fix(X3-282787): fix syracuse unattended installation

### PR DESCRIPTION
**Pull request by lukasz-gruner-sage**
changed CertManagerDataValidator to only set variables when generating new certificate, when copying from mongo installation leave them as-is.
added fallback's for izpack4 - var modify.izpack.install changed case in new version, we read both to support upgrading from izpack 4 -> 5
MongoDBDataValidator always throws warning about "syracuse" db already existing, its ok to ignore it during unattended install

### CODE REVIEW

File: izPackCustomActions/src/com/sage/izpack/PortDataValidator.java [hunk 1]
```diff
@@ -25,7 +25,9 @@ public Status validateData(InstallData adata) {
 		// if update mode
 		// load old value
 
-		boolean updatemode = ("true".equalsIgnoreCase(adata.getVariable("MODIFY.IZPACK.INSTALL")));
+		// izpack5 changed this to low case, we read both to be compatible with version 4 & 5
+		boolean updatemode = ("true".equalsIgnoreCase(adata.getVariable(InstallData.MODIFY_INSTALLATION.toUpperCase()))
+				|| "true".equalsIgnoreCase(adata.getVariable(InstallData.MODIFY_INSTALLATION)));
```
Comment (FranckDepoortere) [inline izPackCustomActions/src/com/sage/izpack/PortDataValidator.java:30]: Maybe create a central method to do that ?

**Review by FranckDepoortere — APPROVED** [review]
Let do that. We will see if there is no side effect during the tests.

File: izPackCustomActions/src/com/sage/izpack/PortDataValidator.java (hunk 1 above)
Comment (lukasz-gruner-sage) [reply izPackCustomActions/src/com/sage/izpack/PortDataValidator.java]: done, I also refactored old classes to use the new Util

**Review by FranckDepoortere — APPROVED** [review]
Looks OK, thanks

**Pull request merged**